config/rewrites.json

```diff
@@ -153,5 +153,147 @@
             "rewrite": false,
             "status": 302
         }
+    ],
+    "api.rackspace.com": [
+        {  "description": "Redirect root traffic to developer.rs.com/docs/",
+            "from": "^\\/$",
+            "to": "/docs/",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Servers redirect",
+            "from": "^\\/api-ref.html$",
```

Comment:
It looks like this route is also the [root of api.rackspace.com](http://api.rackspace.com/) and served at [`/api-ref` (without the .html)](http://api.rackspace.com/api-ref). To adjust the pattern to catch these cases, too, it should be:

```
"from": "^\\/(api-ref(.html)?)?$"
```